diff --git a/packages/opencode/src/cli/cmd/tui/app.tsx b/packages/opencode/src/cli/cmd/tui/app.tsx
index 33be73ca..99748794 100644
--- a/packages/opencode/src/cli/cmd/tui/app.tsx
+++ b/packages/opencode/src/cli/cmd/tui/app.tsx
@@ -484,7 +484,7 @@ function App() {
             <text fg={theme.textMuted}>{process.cwd().replace(Global.Path.home, "~")}</text>
           </box>
         </box>
-        <Show when={false}>
+        <Show when={true}>
           <box flexDirection="row" flexShrink={0}>
             <text fg={theme.textMuted} paddingRight={1}>
               tab
