diff --git a/packages/opencode/src/cli/cmd/tui/app.tsx b/packages/opencode/src/cli/cmd/tui/app.tsx
index 7c72274a..63674b78 100644
--- a/packages/opencode/src/cli/cmd/tui/app.tsx
+++ b/packages/opencode/src/cli/cmd/tui/app.tsx
@@ -484,7 +484,7 @@ function App() {
             </text>
           </box>
         </box>
-        <Show when={false}>
+        <Show when={true}>
           <box flexDirection="row" flexShrink={0}>
             <text fg={theme.textMuted} paddingRight={1}>
               tab
